shared/tracker/action.render.desktop.js

```diff
@@ -12,28 +14,45 @@ export default class ActionRender extends Component {
   props: ActionProps & Styled;
 
   render (): ReactElement {
+    if (flags.tracker2) {
+      return this.render2()
+    }
+    return this.renderDefault()
+  }
+
+  renderDefault (): ReactElement {
     const {username, state, loggedIn} = this.props
+    const styles = styles1
 
     if (!loggedIn) {
-      return this.renderLoggedOut()
+      return this.renderLoggedOut(styles)
     } else if (state === checking || !username) {
-      return this.renderPending()
+      return this.renderPending(styles)
     } else if (this.props.state === normal) {
-      return this.renderNormal(username)
+      return this.renderNormal(styles, username)
     } else if (this.props.currentlyFollowing) {
-      return this.renderChanged()
+      return this.renderChanged(styles)
     } else {
-      return this.renderWarningNotFollowed()
+      return this.renderWarningNotFollowed(styles)
     }
   }
 
-  renderPending (): ReactElement {
+  render2 (): ReactElement {
+    const styles = styles2
+    return (
+      <div style={{...styles.container}}>
+        <FlatButton label='Close' onClick={() => this.props.onClose()} />
```

Comment:
Take a look at `shared/common-adapters/index.js.flow` Those are the common things we should use instead of reaching for material-ui.

So here you should do:
```
import {Button} from '../shared/common-adapters' 
...
<Button label='Close' ... />
```

Also has the added benefit of having all props be typed, so you know if you are missing anything: (https://github.com/keybase/client/blob/master/shared/common-adapters/button.js.flow)